src/components/2019/year-in-review/Opinion.js

```diff
@@ -0,0 +1,170 @@
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    p {
+        margin-bottom: 0px;
+        margin-left: 16px;
+        color: #8E929F;
+        font-weight: bold;
+        font-size: 14px;
+    }
+
+    p:hover {
+        color: #ABAFBA;
+    }
+`
+
+const OpinionTitle = s.h4`
+    color:#131313;
+    ${DM_SERIF_DISPLAY_REGULAR}
+    font-size: 28px;
+    margin-bottom: 12px;
+    
+    :hover{
+        color: #505050;
+    }
+
+`
+
+const Opinion = () => {
+  return (
+      <Wrapper id="opinion">
+          <div class="container">
+            <SectionHeader>Opinion</SectionHeader>
+            <SectionDescription>
+                This year, our columnists reacted to some of the most pressing issues affecting Penn students and the
+                    Philadelphia area. Here are some of the year’s strongest pieces.
+            </SectionDescription>
+            <div class="row">
+                <div class="col-md-4">
+                    <OpinionCard>
+                        <OpinionCardImg className="img-fluid" src="https://snworksceo.imgix.net/dpn/19aff961-4141-4811-bb5b-86d4e8910a9b.sized-1000x1000.jpg?w=1000"/>
+                        <ColumnTitle>UROOBA UNPLUGGED</ColumnTitle>
+                        <a href="https://www.thedp.com/article/2019/02/public-school-ivy-league-children-alumni-urooba-abid-upenn-philadelphia"><OpinionTitle>All Penn alumni should send their kids to public schools</OpinionTitle></a>
+                        <p>Penn graduates sending their kids to private schools is an inherently selfish decision</p>
+                        <AuthorImg>
+                            <OpinionImg
+                                style={{backgroundImage: "url(https://snworksceo.imgix.net/dpn/416973f1-cfe6-4915-ad58-a2f69c16a950.sized-1000x1000.jpg?w=1000)"}}>
+                            </OpinionImg>
+                            <a href="https://www.thedp.com/staff/urooba-abid">
+                                <p class>UROOBA ABID</p>
+                            </a>
+                        </AuthorImg>
+                    </OpinionCard>
+                </div>
+                <div class="col-md-4">
+                    <OpinionCard>
+                        <OpinionCardImg className="img-fluid" src="https://snworksceo.imgix.net/dpn/6a305591-bdf8-4a0c-9e5e-7e3eb1c9cf30.sized-1000x1000.jpg?w=1000"/>
+                        <ColumnTitle>THE OXFORD C'MON</ColumnTitle>
+                        <a href="https://www.thedp.com/article/2019/01/gillette-ad-metoo-masculinity-sophia-durose-upenn-philadelphia"><OpinionTitle>Toxic masculinity runs deeper than the Gillette ad</OpinionTitle></a>
+                        <p>What Penn can learn from the Gillette ad</p>
+                        <AuthorImg>
+                            <OpinionImg
+                                style={{backgroundImage: "url(https://snworksceo.imgix.net/dpn/debec73b-b048-4d5c-90fa-1892021d2c37.sized-1000x1000.png?w=1000)"}}>
```

Comment:
make this a prop that the styled-component takes instead of applying an inline style